# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../dietlibc/maxsymlinks.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

busybox 1.9.1 needs MAXSYMLINKS here

--- ./include/sys/param.h.orig	2008-02-14 14:26:14.000000000 +0000
+++ ./include/sys/param.h	2008-02-14 14:27:30.000000000 +0000
@@ -3,6 +3,7 @@
 
 #include <limits.h>
 
+#define MAXSYMLINKS	20
 #define MAXPATHLEN	PATH_MAX
 #define MAXHOSTNAMELEN	64
 #define NGROUPS		32
